doc/guides/writing-tests.md

```diff
@@ -18,6 +18,13 @@ Add tests when:
 - Fixing regressions and bugs.
 - Expanding test coverage.
 
+## Test directory structure
+See the test [directory structure overview](https://github.com/nodejs/node/blob/master/test/README.md)
+for an outline of existing test types and their locations.
+When deciding on whether to expand an existing test file or create a new one, 
+consider going through the files related to the subsystem (e.g. ones starting with "test-streams" for `lib/streams.js`) 
```

Comment:
Nit: can you wrap this to 80-chars? Also maybe ``ones starting with `test-streams-` for `lib/streams.js` ``

Reply:
fixing it

Reply:
@joyeecheung fixed. Kindly have a look